Packs/CommonScripts/ReleaseNotes/1_2_62.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### New: RelativeDateCheck
+- Evaluates to True if the left value is sooner than the right date.
```

Comment:
```suggestion
- Added a new filter which checks that given time has happend before relative time.
```

Reply:
This should be _after_ not before. I will update.